docs/best_practices/integration_checklist.rst

```diff
@@ -32,6 +32,9 @@ Creating the Tool Wrapper
     - Fill test-data directory with at least one input file and the expected
       output file. The primary output is a good choice for testing. Don't
       forget the use of ``sim_size`` if variable data is included.
+    - It is strongly encouraged that you use small test data sets, ideally
+      under 10 Mb. Every galaxy instance that downloads your tool will
```

Comment:
lets say 1MB for `a` data set max and if they are to big they should use `contains`